### ISSUE TITLE
@Manual annotation is not working - junit

### ISSUE THREAD

**Issue by robyc1982** [serenity-bdd/serenity-core#418]
Hi guys,

I'm trying to use @Manual annotation in serenity 1.1.31 and it seems not to work. I'm using serenity with JUnit.
The tests that I'm trying to annotate are shown in the HTML report as Pending instead of Manual.

I'm trying something like:
@Test
@Manual
public void thisIsAManualTest() {}

Thanks,
Robert.

**Comment by wakaleo** [comment]
Hi Robert,

Could you try with 1.1.34?

**Comment by robyc1982** [comment]
Hi John,
I've tried also with 1.1.34 and now the test annotated with @Manual shows in the HTML report as Automated -> Pass.

Attached a screenshot.

Thanks,
Rob.

[Screen Shot 2016-05-24 at 13.50.12.pdf](https://github.com/serenity-bdd/serenity-core/files/279303/Screen.Shot.2016-05-24.at.13.50.12.pdf)

**Comment by wakaleo** [comment]
Hi Robert,

I'm not seeing this behaviour - you should be seeing something like the one displayed here:
<img width="1008" alt="screen shot 2016-05-24 at 2 58 21 pm" src="https://cloud.githubusercontent.com/assets/55986/15506548/fedacfc4-21bf-11e6-8977-6946cd1ac5fa.png">

Could you check that both the core and the maven plugin are using version 1.1.34?

**Comment by robyc1982** [comment]
I've checked and I have all dependencies upgraded to the latest version: 1.1.34.

I'm also attaching my sample code below.
After running this test using "mvn clean test ... " I'm running "serenity:aggregate" and the generated report shows this manual test as automated and pass, instead of Manual.


@RunWith(SerenityParameterizedRunner.class)
@Story(Authentication.Logout.class)
public class LoginLogoutTest  {

    private String param1;
    private String param2;
    private String param3;

    @TestData(columnNames = "param1, param2, param3")
    public static Collection<Object[]> testData() {
        return Arrays.asList(mainTestData.getTestData());
    }

    public LoginLogoutTest(String param1, String param2, String param3) {
        this.param2 = param2;
        this.param1 = param1;
        this.param3 = param3;
    }

    @Test
    @Manual
    public void thisIsAManualTest() {}

}

**Comment by wakaleo** [comment]
OK, thanks. This looks like a specific issue with the SerenityParameterizedRunner.class runner.

**Comment by aquadrehz** [comment]
I use serenity with cucumber-jvm and seem @Manual is not working too.
Note - selectProduct called from parent Cucumber step definition class. (Contain @Give, @And and etc.)

```
public class CustomerSteps extends ScenarioSteps{
	@Step @Manual
	public void selectProduct(String productName) {
		somepage.selectProduct(productName);
	}
}
```

**Comment by aquadrehz** [comment]
I use serenity with cucumber-jvm and seem @Manual is not working too.
Note - selectProduct called from parent Cucumber step definition class. (Contain @Give, @And and etc.)
- serenity.version = 1.1.34
- serenity.maven.version = 1.1.34
- serenity.cucumber.version = 1.1.8

```
public class CustomerSteps extends ScenarioSteps{
	@Step @Manual
	public void selectProduct(String productName) {
		somepage.selectProduct(productName);
	}
}
```

**Comment by wakaleo** [comment]
This is a documentation issue - the @Manual annotation is not designed to be defined for an individual step within an (otherwise automated) test, but only for the whole test. So if you are using Cucumber, you just put a @Manual tag for the scenario you want to flag as manual.

**Comment by wakaleo** [comment]
This is the intended behaviour - marking a @Step as @Manual is not supported.

**Comment by robyc1982** [comment]
Hi John, 
In my original example, the one for which I've opened this bug for, @Manual annotation is at the Test level. And it's not working with SerenityParameterizedRunner.

Is this issue fixed now? 
I saw you closed the bug. 
If it's not fixed can we reopen it?

Thanks,
Robert.

**Comment by wakaleo** [comment]
Sorry, I didn't mean to close the original bug, which is still present in the Parameterised runner.

**Comment by aquadrehz** [comment]
Test add @Manual to JUnit test scenario.
Still found this issue on last commit

![serenity-418 manaulannotationnotworkinjunit](https://cloud.githubusercontent.com/assets/3850078/16443775/d2b50e44-3e04-11e6-897e-b3333bb36b21.png)

**Comment by wakaleo** [comment]
Fixed in 1.2.2-x
